package.json

```diff
@@ -113,17 +102,7 @@
   },
   "pnpm": {
     "patchedDependencies": {
-      "@types/testing-library__jest-dom@5.14.5": "patches/@types__testing-library__jest-dom@5.14.5.patch",
-      "tsup@7.2.0": "patches/tsup@7.2.0.patch"
-    },
-    "overrides": {
-      "@tanstack/form-core": "workspace:*",
-      "@tanstack/react-form": "workspace:*",
-      "@tanstack/vue-form": "workspace:*",
-      "@tanstack/solid-form": "workspace:*",
-      "@tanstack/yup-form-adapter": "workspace:*",
-      "@tanstack/zod-form-adapter": "workspace:*",
-      "@tanstack/valibot-form-adapter": "workspace:*"
```

Comment:
These overrides are required for the examples to use the local version of the packages to avoid breakage

Reply:
The `prefer-workspace-packages=true` option prevents this :)

Reply:
Oh! Neat, TIL